app/components/navigationComponents/LogoutButton/index.js

```diff
@@ -1,15 +1,51 @@
-import React from 'react';
-import { Image, TouchableOpacity } from 'react-native';
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Image, TouchableOpacity, Alert } from 'react-native';
+import { connect } from 'react-redux';
+import { ROUTES as Routes } from '@config/screens';
+import actionCreators from '@redux/Login/actions';
 
 import img from './assets/user.png';
 import { styles } from './styles';
 
-function LogoutButton() {
-  return (
-    <TouchableOpacity>
-      <Image style={styles.image} source={img} />
-    </TouchableOpacity>
-  );
+class LogoutButton extends PureComponent {
+  logout = () => {
+    const { navigation, logout } = this.props;
+    logout();
+    navigation.navigate(Routes.LOGIN);
+  };
+
+  handleOnPress = () => {
+    Alert.alert('Atención!', '¿Desea cerrar sesión?', [
+      {
+        text: 'Si',
+        onPress: this.logout
+      },
+      {
+        text: 'No'
+      }
+    ]);
+  };
```

Comment:
don't ask, just logout. Unless you make a corresponding modal, we never suggest on using Alerts